packages/cli/src/init/init.js

```diff
@@ -50,6 +50,7 @@ function init(projectDir, argsOrName) {
  * @param options Command line arguments parsed by minimist.
  */
 function generateProject(destinationRoot, newProjectName, options) {
+  // eslint-disable-next-line
   const reactNativePackageJson = require('react-native/package.json');
```

Comment:
This is not run by our CLI, but by `global` CLI and we don't have context here. Ideally, this should be moved out to a sep. location to indicate it's not a command.